web/api/v1/api.go

```diff
@@ -1266,10 +1266,23 @@ func marshalPointJSON(ptr unsafe.Pointer, stream *jsoniter.Stream) {
 	}
 	stream.WriteMore()
 	stream.WriteRaw(`"`)
-	stream.WriteFloat64(p.V)
+
+	// Taken from https://github.com/json-iterator/go/blob/master/stream_float.go#L71 as a workaround
```

Comment:
Is this tested properly in `api_test.go`?

Reply:
I find your lack of faith... disturbing: https://github.com/prometheus/prometheus/blob/76769d4b617989c6c60f61a3ba16261f812538c2/web/api/v1/api_test.go#L1705

Reply:
:cry: I am too lazy